src/Database/LSMTree/Internal/MergeSchedule.hs

```diff
@@ -389,6 +388,7 @@ iforLevelM_ lvls k = V.iforM_ lvls $ \i lvl -> k (LevelNo (i + 1)) lvl
 -- * not stored in snapshots
 -- * not loaded from snapshots
 -- * ignored in lookups
+-- * never merged (by supplying credits to it)
```

Comment:
Maybe:
```suggestion
-- * never completed (by supplying credits to it)
```

Reply:
I don't just mean full completion, but doing any merging work in the tree. So maybe "progressed"?